app/components/Form/SelectInput.tsx

```diff
@@ -126,14 +106,13 @@ function SelectInput({
           onBlur={() => onBlur(value)}
           value={value}
           isValidNewOption={isValidNewOption}
-          shouldKeyDownEventCreateNewOption={shouldKeyDownEventCreateNewOption}
           options={options}
           isLoading={fetching}
           styles={selectStyle ?? selectStyles}
           theme={selectTheme}
           onInputChange={(value) => {
-            if (props.onSearch) {
-              props.onSearch(value);
+            if (onSearch) {
+              onSearch(value);
             }
```

Comment:
Is option chaining better (faster) here?
```suggestion
            onSearch?.(value)
```

Same for below (line 137)